src/devices/bus/coco/coco_ssc.cpp

```diff
@@ -223,6 +225,9 @@ void coco_ssc_device::device_start()
 	save_item(NAME(m_tms7000_portb));
 	save_item(NAME(m_tms7000_portc));
 	save_item(NAME(m_tms7000_portd));
+	
+	m_timer = timer_alloc(0);
```

Comment:
Please use a named constant for the timer ID.  Conflicts are bad enough *with* names, let alone without.